packages/webpack-config-terra/webpack.config.js

```diff
@@ -0,0 +1,250 @@
+const Autoprefixer = require('autoprefixer');
+const PostCSSAssetsPlugin = require('postcss-assets-webpack-plugin');
+const PostCSSCustomProperties = require('postcss-custom-properties');
+const path = require('path');
+const rtl = require('postcss-rtl');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+const DuplicatePackageCheckerPlugin = require('@cerner/duplicate-package-checker-webpack-plugin');
+const aggregateTranslations = require('terra-aggregate-translations');
+const ThemePlugin = require('./lib/postcss/ThemePlugin');
+const getThemeConfig = require('./lib/utils/_getThemeConfig');
+
+/**
+ * Get the locales to be defined by the webpack build, also aggregates translations, but we should remove that in the future.
+ * @param {object} env the env object
+ * @param {string} rootPath root path of the package.
+ */
+const getLocales = ({ disableAggregateTranslations }, rootPath) => {
+  if (!disableAggregateTranslations) {
+    return aggregateTranslations({ baseDir: rootPath });
+  }
+  return undefined;
+};
+
+/**
+ * Get an array of modules that need to be resolved to during the webpack build. Should be removed after aggregate translations is removed.
+ * @param {object} env the env object
+ */
+const getResolveModules = ({ disableAggregateTranslations }) => (
+  [
+    'node_modules',
+    ...disableAggregateTranslations ? [] : ['aggregated-translations'],
+  ]
+);
+
+/**
+ * Determines the correct theme config to usee of the three options. Precedence is. ENV, options config, terra-them.config.js
+ * @param {object} options object containing the override config if provided.
+ */
+const determineThemeConfig = ({ themeConfig: overrideThemeConfig }) => {
+  const defaultTheme = process.env.THEME; // Flexes root theme for theme visual regression testing.
```

Comment:
in our wdio setup, we will execute the wdio function when we compile. Can we pass this as a webpack env instead of a proces.env?

https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/webpack-server/webpack-server.js#L28

Reply:
Yes i think we could pas this as a .env. I didn't update this yet because I wanted to talk to folks about it's ramifications. primarily you have to figure out how to chain down the webpack env on npm commands instead of just setting it at the beginning of the command.

Reply:
I've made this change, but passively, we still support THEME env until next mvb and now we support the defaultTheme webpack env option